R/chi_check.R

```diff
@@ -31,6 +31,11 @@
 #' * Is the checksum digit correct?
 #'
 #' @param x a CHI number or a vector of CHI numbers with `character` class.
+#' @param check_mod11 A boolean (default `TRUE`). Check CHI numbers using Modulo 11.
+#' @param check_mod10 A boolean (default `TRUE`). Check CHI numbers using Modulo 10.
```

Comment:
You can group related params in the help page like this: `#' @param check_mod11,check_mod10`.

I think what I've written below is sensible and correct but at least take the gist of it 😆 
```suggestion
#' @param check_mod11,check_mod10 Logical values (TRUE or FALSE, default is `TRUE`). By default, a CHI that passes either the modulo 10 or the modulo 11 check will be considered valid. Historically, CHIs only used modulo 11 for their check digit; however, starting from 2026, some CHIs will only pass if they meet the modulo 10 criteria.
```